server/controllers/updateEvent.js

```diff
@@ -0,0 +1,7 @@
+const { Event } = require('../database/models');
+
+exports.updateEvent = (req, res) => {
+  Event.update({ status: 1 }, { where: { id: req.params.id } })
+    .then(() => res.json({ success: 'status updated' }))
```

Comment:
Check if there is no result 'do something' (send a message to front-end), it's a way to avoid crash server

Reply:
done